### PR TITLE
Added support for Redaxo V 5.*

### PR DESCRIPTION
**Pull request by TheFehr**
I added new redaxo5-addon and redaxo5-bestyle-plugin types.

### CODE REVIEW

**Comment by TheFehr** [comment]
Will this be added or should I close this pull request?

**Comment by TheFehr** [comment]
I just reopened this pull request to see if travis still fails. And it did not. So why has this not yet been merged? The lack of response is a bit sad.

**Comment by dajoho** [comment]
I had forgotten completely that I ever made this. I have since left the REDAXO scene and only install it now again to see the current state of affairs. Thanks for finishing the work! :-)

**Comment by TheFehr** [comment]
@alcohol how do you decide on which pr's to merge? How can I improve this one?

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ is not needed to install packages with these frameworks:
 | Porto        | `porto-container`
 | RadPHP       | `radphp-bundle`
 | REDAXO       | `redaxo-addon`
+| REDAXO V5.*  | `redaxo5-addon`
```
Comment (niksamokhvalov) [inline README.md]: You may need to add `redaxo5-bestyle-plugin`?

Comment (TheFehr) [reply README.md]: I also added the missing redaxo-bestyle-plugin

**Comment by TheFehr** [comment]
@niksamokhvalov I added the bestyle plugin intregration for both versions into the readme. How else can I improve this to get it merged?

**Pull request merged**